cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java

```diff
@@ -36,49 +34,13 @@
         t3xMapping = new HashMap<>();
     }
 
-    public void add(String transformerId, String rtcId, PropertyBag rtc, PropertyBag ptc) {
```

Comment:
Was this method never used?

Reply:
Is not used. It was used in the old conversion, deleted months ago. Methods used now are  `add(String transformerId, CgmesRegulatingControlRatio rcRtc, CgmesRegulatingControlPhase rcPtc)` and  `
add(String transformerId, CgmesRegulatingControlRatio rcRtc1, CgmesRegulatingControlPhase rcPtc1,
                    CgmesRegulatingControlRatio rcRtc2, CgmesRegulatingControlPhase rcPtc2, CgmesRegulatingControlRatio rcRtc3,
                    CgmesRegulatingControlPhase rcPtc3)
`